.github/workflows/build.yaml

```diff
@@ -32,6 +32,9 @@ jobs:
           java-version: ${{ matrix.jdk }}
           distribution: 'temurin'
           cache: 'maven'
+      #- uses: actions-rust-lang/setup-rust-toolchain@v1
+      - run: ./update-rust-jni-libs.sh
```

Comment:
todo: we should publish these libs in a way that it's available for the release build of the maven lib.

or maybe just run maven package on a osx runner.